src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryString = string;
+  const salaryNumber = salaryString.replace('$', '').replace(',', '');
+
+  return +salaryNumber;
+}
```

Comment:
It's a good practice to avoid mutation of function arguments. In this case, you don't really need the `salaryString` variable. You can directly use the `string` argument in the `replace` method.